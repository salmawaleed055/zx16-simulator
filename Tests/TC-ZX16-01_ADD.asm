li x1, 5
li x2, 3
add x1, x2
ecall 0x3FF