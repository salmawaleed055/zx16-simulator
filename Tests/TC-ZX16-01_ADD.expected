Loaded 8 bytes into memory
0x0000: 0A79    li x1, 5
0x0002: 06B9    li x2, 3
0x0004: 0440    add x1, x2
0x0006: FFC7    Unknown opcode

--- Final State ---
x0: 0x0000
x1: 0x0008
x2: 0x0003
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x0006
---------------------

Simulation finished.
